src/device/router_device.erl

```diff
@@ -58,6 +59,7 @@
 ]).
 
 -define(QUEUE_SIZE_LIMIT, 20).
+-define(MAX_32_BITS, 16#100000000).
```

Comment:
This comes out to 33bits.

There's a util for this, `router_device_devaddr:uint32/1`, might be a good opportunity to move it to `router_utils`.